packages/db/tests/models/test-granule-model.js

```diff
@@ -652,10 +1791,28 @@ test('GranulePgModel.upsert() succeeds without an execution for completed granul
     collection_cumulus_id: collectionCumulusId,
   });
 
-  await granulePgModel.upsert(knex, granule);
+  await granulePgModel.upsert({ knexOrTrx: knex, granule });
   t.true(await granulePgModel.exists(knex, granule));
 });
 
+test('GranulePgModel.upsert() succeeds for a granule without createdAt set and sets a default value', async (t) => {
```

Comment:
do we need to delete `createdAt` on the granule object?

Reply:
An excellent point - and in fact this test title doesn't make sense out of the context of the original PR with the changed logic.  We're missing a couple of tests here and this one isn't specific enough: 

59c95dcb5f